src/react-components/src/components/03-organisms/Headers/HeaderWithSearch.stories.tsx

```diff
@@ -0,0 +1,47 @@
+/* OH-31 Header With Search */
+import * as React from "react";
+
+import HeaderWithSearch from "./HeaderWithSearch";
+import { action } from "@storybook/addon-actions";
+import { withKnobs, text } from "@storybook/addon-knobs";
+import UnderlineLink from "../../01-atoms/Links/UnderlineLink";
+
+
+export default {
+  title: "HeaderWithSearch",
+  component: HeaderWithSearch,
+};
+
+export const researchNowSearchHeader = () => <HeaderWithSearch
+  searchButtonId="searchButtonId"
+  searchBarAriaLabel="Search research catalog"
+  advancedSearchElem={<UnderlineLink url={"advanced-search-link"}>Advanced Search</UnderlineLink>}
+  searchBarId="searchBarId"
+  dropdownId="dropdownId"
+  textFieldAriaLabel="Research Now"
+  headingContent={<span>Research<span className={"rn-section-title__emphasis"}>Now</span></span>}
```

Comment:
Adding this prop doesn't do anything. It's declared but not being used in `HeaderWithSearch`.

Reply:
I'm okay either way, but eventually we'll want this to accept a generic header, not just the RN specific one.  It's just a variation that hasn't been implemented yet.

Reply:
Makes sense but I don't think it should be included as part of an example if it's not working and will be implemented later. It will bring confusion for _this_ release, although for a future implementation it won't be a problem.

Reply:
Makes sense, it's not a big deal to put this kind of stuff back in in the future!